jbpm-workitems/jbpm-workitems-repository-archetype/README.md

```diff
@@ -0,0 +1,42 @@
+# WorkItem Archetype
+
+Archetype used to build new jBPM Workitems.
+
+This Archetype allows you to easily start building a new jBPM Workitem and easily be contributed to the workitem repository.
+It creates your WorkItem Handler Maven project that includes the base Handler class, test, and
+a zip file that includes all needed files to upload to a jBPM workitem repository.
+
+How to run it
+--------------------
+1. build the jbpm-workitems module (mvn clean install)
+2. Change to directory of your choice where you want to build the 
+base workitem handler from this archetype.
+3. Create your new workitem handler from the archetype with:
+```
+mvn archetype:generate 
+   -DarchetypeGroupId=org.jbpm 
+   -DarchetypeArtifactId=jbpm-workitems-repository-archetype
+   -DarchetypeVersion=7.38.0-SNAPSHOT
```

Comment:
keep this versions in sync with pom - I think they are being replaced during upgrade? @mbiarnes

Reply:
@bxf12315 please put there the current version, it's not replaced automatically, but manually by full-text searching for the current version. So if we're not in sync with current version it will stay not-updated.

Reply:
Done